.travis.yml

```diff
@@ -51,11 +51,15 @@ jobs:
       env:
         - "INSTALL_METHOD=none"
       name: KinD (k8s-1.19)
+      before_script:
+        - docker login -u "$REGISTRY_USER" -p "$REGISTRY_PASS"
       script: ./test/e2e.sh $INSTALL_METHOD
     - stage: deploy
       go: "1.14"
       sudo: required
       name: Publish Container Image
+      before_script:
+        - docker login -u "$REGISTRY_USER" -p "$REGISTRY_PASS"
```

Comment:
If we log in here, the log in `before_deploy` (below line #67) can be removed.

Reply:
Trying it now...